module/icp/algs/skein/skein_block.c

```diff
@@ -239,8 +238,7 @@ Skein_256_Process_Block(Skein_256_Ctxt_t *ctx, const uint8_t *blkPtr,
 		Skein_Show_Round(BLK_BITS, &ctx->h, SKEIN_RND_FEED_FWD, ctx->X);
 
 		ts[1] &= ~SKEIN_T1_FLAG_FIRST;
-	}
-	while (--blkCnt);
+	}	while (--blkCnt);
```

Comment:
Just a single space.  This passes cleanly for me and it's the expected formatting for a do/while construct.

```diff
@@ -238,7 +238,8 @@ Skein_256_Process_Block(Skein_256_Ctxt_t *ctx, const uint8_t
                Skein_Show_Round(BLK_BITS, &ctx->h, SKEIN_RND_FEED_FWD, ctx->X);
 
                ts[1] &= ~SKEIN_T1_FLAG_FIRST;
-       }       while (--blkCnt);
+       } while (--blkCnt);
+
        ctx->h.T[0] = ts[0];
        ctx->h.T[1] = ts[1];
 }
```